src/main/java/seedu/address/logic/Messages.java

```diff
@@ -15,6 +15,7 @@ public class Messages {
     public static final String MESSAGE_UNKNOWN_COMMAND = "Unknown command";
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
     public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "The person index provided is invalid";
+    public static final String MESSAGE_INVALID_PERSON = "The ic number provided is invalid";
```

Comment:
#47 has a similar message. I think we can leave both in for now, but we might either want to refactor them together into one constant in the future, or make a separate messages for each command.